website/docs/reference/function-configs.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Function configurations
+description: "Read this guide to learn about using function configurations in dbt."
+meta:
+  resource_type: Functions
+---
+
+import ConfigResource from '/snippets/_config-description-resource.md';
+import ConfigGeneral from '/snippets/_config-description-general.md';
+
+
+## Available configurations
+### Function-specific configurations
+
+<ConfigResource meta={frontMatter.meta} />
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+  ]
+}>
+<TabItem value="project-yaml">
+
+<File name='dbt_project.yml'>
+
+```yml
+functions:
```

Comment:
should we update https://docs.getdbt.com/reference/dbt_project.yml too then?
<img width="328" height="449" alt="Screenshot 2025-10-08 at 6 35 33 AM" src="https://github.com/user-attachments/assets/576e1fc0-ef97-432d-85ed-9904fc5f4279" />

Reply:
also - @QMalcolm do we have a `function-paths` dbt_project.yml property to optionally specify a custom list of directories where functions are located?